hu.json

```diff
@@ -0,0 +1,190 @@
+﻿{
+    "name" : "Magyar",
+    "live" : true,
+    "translations" : {
+        "meta" : {
+            "thForDate" : true,
+            "dateFormat" : "[year] [month] [day]",
+            "months" : [
+                "Január",
+                "Február",
+                "Március",
+                "Április",
+                "Május",
+                "Június",
+                "Július",
+                "Augusztus",
+                "Szeptember",
+                "Október",
+                "November",
+                "December"
+            ],
+            "types" : {
+                "backpack" : "Hátizsák",
+                "emote" : "Emote",
+                "glider" : "Siklórepülő",
+                "emoji" : "Hangulatjel",
+                "loading" : "Töltő képernyő",
+                "outfit" : "Ruházat",
+                "pickaxe" : "Csákány",
+                "skydive" : "Zuhanási nyomvonal",
+                "umbrella" : "Esernyő",
+                "spray" : "Spré",
+                "toy" : "Játék",
+                "pet" : "Háziállat",
+                "music" : "Zene",
+                "wrap" : "Csomagolópapír",
+                "bundle" : "Csomag",
+                "misc" : "Vegyes"
+            },
+            "rarity" : {
+                "legendary" : "Legendás",
+                "epic" : "Epik",
+                "rare" : "Ritka",
+                "uncommon" : "Rendkívüli",
+                "common" : "Közönséges"
+            },
+            "translationNotAvailable" : "Sajnos nincsen elérhető fordítás erre az oldalra.",
+            "translationIncomplete" : "Ezen az oldalon nem az összes szöveg lesz lefordítva."
+        },
+        "nav" : {
+            "home" : "főoldal",
+            "shop" : "bolt",
+            "cosmetics" : {
+                "dropdown" : "kinézetek",
+                "upcoming" : "közelgő",
+                "list" : "lista",
+                "png" : "png",
+                "gallery" : "galléria",
+                "icons" : "ikonok",
+                "reminders" : "emlékeztetők",
+                "history" : "bolt történelem"
+            },
+            "news" : "hírek",
+            "modes" : "módok",
+            "randomiser" : "véletlenszerűsítő",
+            "about" : {
+                "dropdown" : "about",
+                "about" : "fnbr.co-ról",
+                "privacy" : "Adatvédelmi irányelvek",
+                "donate" : "Adományozz",
+                "discord" : "Discord",
+                "twitter" : "Twitter"
+            },
+            "api" : "api",
+            "account" : {
+                "dropdown" : "Fiók",
+                "account" : "Fiók",
+                "logout" : "Kijelentkezés"
+            },
+            "languagePicker" : {
+                "trigger" : "Nyelv",
+                "title" : "Válassz ki egy nyelvet.",
+                "currentLanguage" : "Te most a [Magyar] nyelvet használod."
```

Comment:
Can you change the text within [] back to [language] it is needed for the language to be filled in automatically